Add chapter about PRQL (#1300)
ClickHouse/clickhouse-docs: docs/en/tutorial.md

    LIMIT 1000
    ```

## 6. Alternative Query Languages

> You can other query languages to query data in ClickHouse using the `dialect` setting. The currently supported dialects are the followings:

Reviewer: ```suggestion
You can use other query languages to query data in ClickHouse using the `dialect` setting. The currently supported dialects are:
```

- `clickhouse`: The default [ClickHouse SQL dialect](./sql-reference/syntax.md)
- `prql`: [Pipelined Relational Query Language](https://prql-lang.org/)

You can execute queries using the PRQL after setting the dialect to `prql`:
```sql
SET dialect = 'prql'
```

Then you can use every PRQL feature that the included PRQL compiler supports:

```prql
from trips
aggregate [
    ct = count
]
```

> Under the hood ClickHouse will translate the PRQL query into an SQL query and execute it. Once you run all the queries you wanted, you can switch back to the ClickHouse SQL dialect by setting the dialect to `clickhouse`:

Reviewer: ```suggestion
Under the hood ClickHouse will translate the PRQL query into an SQL query and execute it. To switch back to the ClickHouse SQL dialect set the dialect to `clickhouse`:
```
